### ISSUE TITLE
iOS: [a11y] Settings (gear icon) not reachable with VoiceOver

### ISSUE THREAD

**Issue by mlang** [bitwarden/android#339]
The settings (gear) icon in the initial screen (login or create account) is not reachable nor spoken at all by VoiceOver, the screen reader for the blind on iOS.

I can swipe between three elements on that initial screen:
* The text "You have to login or regster to access your vault"
* The "login" button
* The "create account" button

Touching the left upper corner of the screen (where the gear icon resides) does not help, voiceover does not speak the settings icon at all, and also does not allow to invoke it.

To test this, you will likely have to enable VoiceOver in Accessibility menu of the iOS Settings application.

**Comment by kspearrin** [comment]
Thanks. I imagine the mobile app as a whole is pretty terrible at a11y. There are plans for a re-write in the future where a11y will be a priority.

**Comment by steverep** [comment]
@kspearrin ,

> I imagine the mobile app as a whole is pretty terrible at a11y. 

Actually, it's really not that bad.  I regularly use it with VoiceOver and work around this issue and the lack of accessible labels as mentioned in #187.

> There are plans for a re-write in the future where a11y will be a priority.

Given it's not that bad to begin with, why not make it a priority now?  A rewrite is certainly not necessary to fix some unlabeled buttons and tabs.

**Comment by vvolkgang** [comment]
Issue migrated to https://github.com/bitwarden/mobile/issues/339
